src/GovUk.Education.ExploreEducationStatistics.Admin/Services/ReleaseService.cs

```diff
@@ -494,9 +492,10 @@ private void CreateGenericContentFromTemplate(Guid releaseId, Release newRelease
                     await _dataBlockService.DeleteDataBlocks(deletePlan.DeleteDataBlockPlan);
                     await _releaseSubjectService.SoftDeleteSubjectOrBreakReleaseLinkAsync(releaseId, deletePlan.SubjectId);
 
-                    return await _fileStorageService
+                    await _fileStorageService
                         .RemoveDataFileReleaseLinkAsync(releaseId, fileName)
-                        .OnSuccessDo(async () => await RemoveFileImportEntryIfOrphaned(deletePlan));
+                        .OnSuccess(async () => await RemoveFileImportEntryIfOrphaned(deletePlan));
```

Comment:
I spotted that this returns `bool`, thinking it could be used as the successful return value of this method, but looks like it returns false if not an orphan?

Reply:
yes I think that was there originally because we still wanted to be able to delete if things were left in an incomplete state for whatever reason but given what we discussed about work required for the atomic handling (or lack of it) I'll bear this in mind when carrying out that work.